shell/platform/android/BUILD.gn

```diff
@@ -89,6 +89,26 @@ shared_library("flutter_shell_native") {
   ldflags = [ "-Wl,--version-script=" + rebase_path("android_exports.lst") ]
 }
 
+action("gen_android_build_config_java") {
+  script = "$flutter_root/tools/gen_android_buildconfig.py"
+
+  build_config_java = "$target_gen_dir/io/flutter/BuildConfig.java"
+
+  outputs = [
+    build_config_java,
+  ]
+
+  args = [
+    "--out",
+    rebase_path(build_config_java),
+    "--runtime-mode",
+    flutter_runtime_mode,
+  ]
+
+  build_config = []
+  assert(build_config == [] || true)
```

Comment:
I dont think you meant to leave this in here.

Reply:
You are correct.